community/cypher/docs/cypher-docs/src/docs/dev/ql/match/index.asciidoc

```diff
@@ -1,33 +1,33 @@
 [[query-match]]
-Match
-=====
+= Match
+
+[abstract]
+The `MATCH` clause is used to search for the pattern described in it.
 
 == Introduction ==
 
-The +MATCH+ clause allows you to specify the patterns Cypher will search for in the database, and
-it is the primary way of getting data into the current set of bindings.
+The `MATCH` clause allows you to specify the patterns Neo4j will search for in the database.
+This is the primary way of getting data into the current set of bindings.
 It is worth reading up more on the specification of the patterns themselves in <<introduction-pattern>>.
 
-It is often coupled to a +WHERE+ clause which adds restrictions, or predicates, to the +MATCH+ patterns, making them more specific.
+`MATCH` is often coupled to a `WHERE` clause which adds restrictions, or predicates, to the `MATCH` patterns, making them more specific.
```

Comment:
@systay  do you think it confuses the mental model the user has if we talk about `MATCH` and `WHERE` as being separate clauses?